KeyboardHeightConstraint.swift

```diff
@@ -0,0 +1,54 @@
+//
```

Comment:
Will you pls move the file to appropriate file directory?

Reply:
Yes, of course.

Reply:
Done.